moto_sync_from_s3/Tiltfile

```diff
@@ -0,0 +1,131 @@
+load("ext://uibutton", "cmd_button", "location")
+
+EXTENSION_ROOT = os.getcwd()
+PROJECT_ROOT = config.main_dir
+
+DOTENV_PREFIX = "S3SYNC"
+DOTENV_REQUIRED = [
+    "AWS_ACCOUNT_ID",
+    "AWS_S3_ROLE_NAME",
+    "AWS_SSO_WEB_START_URL",
+]
+
+
+def moto_sync_from_s3(
+    bucket_name,
+    object_prefix = "/",
+    dotenv_path = ".env",
+    dotenv_prefix = DOTENV_PREFIX,
+    button_location = location.NAV,
+    sync_on_start = False,
+    resource_deps = ["motoserver"],
+    labels = ["motoserver"],
+):
+    """Sync data from remote S3 bucket to its local devenv counterpart."""
+    if not dotenv_prefix:
+        fail("The `dotenv_prefix` argument is incorrect: Please provide a non-empty value or omit to use default!")
+
+    cmd = [os.path.join(EXTENSION_ROOT, "scripts/bootstrap.sh")]
+    cmd.extend([bucket_name, object_prefix])
+    env = _handle_dotenv(dotenv_path, dotenv_prefix, bucket_name)
+
+    resource_name = "sync_s3_%s_%s" % (
```

Comment:
I think you should allow `resource_name` to be customized or set from an env variable. I think this is okay for the default but as it stands the default name will only get longer depending on how long the bucket name and prefix are. This feels alot longer  than id like: `sync_s3_<bucket>_<lengthy_nested_prefix>`